Objective-C/CBLMutableDocument.mm

```diff
@@ -119,6 +119,10 @@ - (CBLMutableDocument*) mutableCopyWithZone: (NSZone *)zone {
     return [[CBLMutableDocument alloc] initAsCopyWithDocument: self dict: _dict];
 }
 
+- (CBLMutableDocument*) copy {
+    return [self mutableCopyWithZone: nil];
```

Comment:
I don't remember if the default copy calls mutableCopyWithZone or not. Can you double check?